src/protocols/bdx/BdxMessages.h

```diff
@@ -169,10 +201,8 @@ struct SendAccept
     uint16_t MaxBlockSize = 0; ///< Chosen max block size to use in transfer (required)
 
     // Additional metadata (optional, TLV format)
-    // WARNING: there is no guarantee at any point that this pointer will point to valid memory.
-    // It is up to the caller to ensure that the memory pointed to here has not been freed.
```

Comment:
I removed it because there is now a PacketBufferHandle to the data included in the struct. I guess there's no guarantee that the PacketBufferHandle points to the buffer but it seems unnecessary to me to include a warning about that.

But if you think it should stay I don't mind adding it back in.

Reply:
It should it should stay, because it's a raw pointer. If you kept reference to it, it would give long term access to disappeared storage.

Reply:
Ok sure. I'll add the warning back in